b2b-sample/guardio-insurance-sample-application/components/settingsComponents/addUserComponent.js

```diff
@@ -0,0 +1,223 @@
+import { errorTypeDialog, successTypeDialog } from '../util/dialog';
+
+import styles from '../../styles/Settings.module.css';
+import decodeAddUser from '../../util/apiDecode/settings/decodeAddUser';
+import SettingsTitle from '../util/settingsTitle';
+import SuccessDialog from '../util/successDialog';
+
+export default function AddUserComponent(props) {
+
+    const ADD_USER_COMPONENT = "ADD USER COMPONENT";
+
+    const [successDialogOpen, setSuccessDialogOpen] = useState(false);
+
+    const [loadingDisplay, setLoadingDisplay] = useState(LOADING_DISPLAY_NONE);
+
+    const toaster = useToaster();
+
+    const nameValidate = (name, errors) => {
+        if (!name) {
+            errors.name = 'This field cannot be empty'
+        }
+        return errors;
+    }
+
+    const emailValidate = (email, errors) => {
+        if (!email) {
+            errors.email = 'This field cannot be empty'
+        }
+        return errors;
+    }
+
+    const usernameValidate = (username, errors) => {
+        if (!username) {
+            errors.username = 'This field cannot be empty'
+        }
+        return errors;
+    }
+
+    const passwordValidate = (password, errors) => {
+        if (!password) {
+            errors.password = 'This field cannot be empty'
+        }
+        return errors;
+    }
+
+    const repasswordValidate = (repassword, errors) => {
+        if (!repassword) {
+            errors.repassword = 'This field cannot be empty'
+        }
+        return errors;
+    }
+
+    const validate = values => {
+        const errors = {}
+        errors = nameValidate(values.name, errors);
+        errors = emailValidate(values.email, errors);
+        errors = usernameValidate(values.username, errors)
+        errors = passwordValidate(values.password, errors);
+        errors = repasswordValidate(values.repassword, errors);
+
+        return errors
+    }
+
+    const onDataSubmit = (response, form) => {
+        if (response) {
+            successTypeDialog(toaster, "Changes Saved Successfully", "User add to the organization successfully.");
+            form.restart();
+        } else {
+            errorTypeDialog(toaster, "Error Occured", "Error occured while adding the user. Try again.");
+        }
+    }
+
+    const onSubmit = async (values, form) => {
+        setLoadingDisplay(LOADING_DISPLAY_BLOCK);
+        decodeAddUser(props.session, values.name, values.email,
+            values.username, values.password)
+            .then((response) => onDataSubmit(response, form))
+            .finally((response) => setLoadingDisplay(LOADING_DISPLAY_NONE))
+    }
+
+    const closeSuccessDialog = () => {
+        setSuccessDialogOpen(false);
+    }
+
+    return (
+        <div className={styles.addUserMainDiv}>
+            <SuccessDialog open={successDialogOpen} onClose={closeSuccessDialog} />
+
+            <SettingsTitle title="Add User" subtitle="Add a new user to the organisation" />
```

Comment:
Spelling mistakes. `Add a new user to the organization`

Reply:
Fixed with https://github.com/Achintha444/samples-is/commit/4120b50f44ba393f00f7c7cf57aa04d21f6bc8d6#diff-5091835390e887659dc8a6ad4bd13ec79e7d83568a32943c52a2edd117bdb012